app/src/main/res/layout/fragment_tips_severity1.xml

```diff
@@ -0,0 +1,229 @@
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/title" />
+
+        <TextView
+            style="@style/Question"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="25dp"
+            android:layout_marginBottom="10dp"
+            android:text="What do I do?" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="20dp"
+            android:orientation="horizontal"
+            tools:ignore="UseCompoundDrawables">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingEnd="16sp"
+                android:src="@drawable/ic_tick"
+                tools:ignore="ContentDescription,RtlSymmetry" />
+
+            <TextView
+                style="@style/Answer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Stay home. Make sure you feel better." />
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="20dp"
+            android:orientation="horizontal"
+            tools:ignore="UseCompoundDrawables">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingEnd="16sp"
+                android:src="@drawable/ic_tick"
+                tools:ignore="ContentDescription,RtlSymmetry" />
+
+            <TextView
+                style="@style/Answer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Don't go shopping or have vistors." />
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            tools:ignore="UseCompoundDrawables">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingEnd="16sp"
+                android:src="@drawable/ic_tick"
+                tools:ignore="ContentDescription,RtlSymmetry" />
+
+            <TextView
+                style="@style/Answer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Ask others to go shopping for you or deliver at home. Ask if someone else can walk the dog." />
```

Comment:
shouldn't we keep the text in the string xml to make it easier for us to localise?

Reply:
yes, you're absolutely right! will change this when I have the final copies of the text